models/klaviyo.yml

```diff
@@ -0,0 +1,276 @@
+version: 2
+
+models:
+  - name: klaviyo__events
```

Comment:
The below columns are not documented within the dbt docs currently.
```txt
- _fivetran_synced
- subject_line
```
Can we add them here before release

Reply:
ah i need to remove `subject_line` -- that's a custom column i was passing through. will add _fivetran_synced tho